amarachukwu-agbo/hello-books
# #158556992 Remove user's id from url

#### What does this PR do?

Remove user's id from the url

#### Description of Task to be completed?
- Modify url not to use user's id
- Group components into folders
- Add jsdoc comments

#### How should this be manually tested?

Clone the repo and install application dependencies. Run command  `npm run start:server` and `npm run start:client`. Open the browser on `127.0.0.1:8080/` and login. Click the `Profile` tab on the nav bar. You should the url does not contain user's id.


#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

#158556992

#### Screenshots (if appropriate)

N/A

#### Questions:
N/A
